frontend/src/services/notebookService.ts

```diff
@@ -37,6 +37,7 @@ export const createNotebook = (
   const url = `/api/notebooks/${projectName}`;
   const resources = { ...notebookSize?.resources };
   const tolerations: NotebookToleration[] = [];
+  let affinity = {};
```

Comment:
@samuelvl Is an empty affinity object okay for the notebook pod if gpus are present?

Reply:
Yes, it's totally fine